app/controllers/articles_controller.rb

```diff
@@ -244,6 +243,16 @@ def set_article
     @article = found_article || not_found
   end
 
+  def set_surrogate_keys_by_case
+    if @tag_model
+      set_surrogate_key_header @tag_model.record_key, Article.table_key, Tag.table_key, @articles.map(&:record_key)
+    elsif @user
+      set_surrogate_key_header @user.record_key, Article.table_key, User.table_key, @articles.map(&:record_key)
+    else
```

Comment:
Do we need a case for organization here?

Reply:
Yep, I think so, orgs have their feed: https://dev.to/feed/devteam